### PR TITLE
add peaklingo hiring link

### PR DESCRIPTION
**Pull request by msadegh76**
add peaklingo hiring announcement to careers page

### CODE REVIEW

File: _data/careers.yml [hunk 1]
```diff
@@ -9,3 +9,9 @@
   apply_link: https://careers.playpod.ir/career/rust-developer/
   logo: /assets/careers/playpod.png
   description: is hiring Rust, C and C++ developers
+
+- name: Peaklingo
+  website:
```
Comment (hamidrezakp) [inline _data/careers.yml:14]: Add your website link here, or `#` in case there is no website accessible.

**Review by hamidrezakp — CHANGES_REQUESTED** [review]
Other than website URL, other things lgtm.

File: _data/careers.yml (hunk 1 above)
Comment (msadegh76) [reply _data/careers.yml:14]: Sure @hamidrezakp 
Thanks

**Review by hamidrezakp — APPROVED** [review]

**Comment by hamidrezakp** [comment]
Thanks 🎉

**Pull request merged**